integrations/docker/images/chip-cert-bins/README.md

```diff
@@ -0,0 +1,46 @@
+# Docker image for Matter Certification Test Harness
+
+The Dockerfile here helps build multi-platform Docker images containing the  executable binaries necessary for the Matter Test Harness. It utilizes the BuildKit toolkit and Buildx included within Docker since version 18.06.
+
+## Running
+
+In order to properly run the binaries, avahi must be properly set up and passed to the container.
+
+Prerequisites:
+
+* Host must support and enable IPv6 and be on a network that has IPv6.
+* IPv6 must be enabled within avahi config on the host. ```use-ipv6=yes``` in avahi-daemon.conf
+* Sometimes there are stale avahi entries, so restarting avahi-daemon between runs may be necessary.
```

Comment:
This seems vague. Why not using minmdns and not install avahi at all?

Reply:
I believe avahi is installed by default on Raspberry Pi and most desktop linux distributions.

Reply:
If minmdns is more stable, we can remove avahi from the Ubuntu installation on the host machine.

What do you think @andy31415

Reply:
The difference will be that avahi has some caching behaviour. Not sure if it matters - I would expect all to work as we have to maintain both.

My comment mostly was `this uses non-default settings` and would like the cause to be documented. I would like to understand more since we need to make minmdns work so if we have a statement of the form `avahi works but minmdns does not` that does not help us improve things.

The only case where I have found avahi working but minmdns not was in cases of firewalls allowing avahi but not minmdns. That should not affect cert (I assume cert does not run on a restrictive firewalled RPI)